src/vorta/utils.py

```diff
@@ -249,10 +268,14 @@ def pretty_bytes(size, metric=True, sign=False, precision=1):
         if metric
         else (2**10, ['', 'Ki', 'Mi', 'Gi', 'Ti', 'Pi', 'Ei', 'Zi', 'Yi'])
     )
-    n = 0
-    while abs(round(size, precision)) >= power and n + 1 < len(units):
-        size /= power
-        n += 1
+    if fixed_unit is None:
+        n = 0
+        while abs(round(size, precision)) >= power and n + 1 < len(units):
+            size /= power
+            n += 1
+    else:
+        n = fixed_unit
+        size /= power**n
     try:
         unit = units[n]
```

Comment:
What if index out of range? See comment above.

Reply:
right, better clamp it